go.mod

```diff
@@ -2,10 +2,15 @@ module github.com/tetratelabs/proxy-wasm-go-sdk
 
 go 1.17
 
-require github.com/stretchr/testify v1.7.0
+require (
+	github.com/stretchr/testify v1.7.0
+	github.com/tidwall/gjson v1.14.0
```

Comment:
I think at least this example should be an own go module to avoid adding unrelated deps to this module. How about that?

Reply:
oh yeah agreed. it's time to have separate gomod for each example. Would you mind doing that in another PR?

Reply:
Done: https://github.com/tetratelabs/proxy-wasm-go-sdk/pull/265